src/utils/constants.js

```diff
@@ -44,17 +44,5 @@ export const ACTIVITY_HISTORY = [
   
 ];
 export const ASSETS_VALUE = [
-  // we can put assets address on the place of assetsname
-
   "0xd0A1E359811322d97991E03f863a0C30C2cF029C",
```

Comment:
can we make ASSETS_VALUE a dict with keys as names of assets and value as their address.
ASSETS_VALUE = {'ETH': 123213213, 'LINK': 3213213}

Reply:
that this is already done in the miner changes branch
just let me know when you are free so that we can work with that

Reply:
this is already done  in one of my branch but rather than using dic it is using an array where index 0 contain ether address and so on for other assets